mhcflurry/class1_affinity_prediction/train_allele_specific_models_command.py

```diff
@@ -85,29 +132,30 @@ def run(argv=sys.argv[1:]):
     for (h, hyperparameters) in enumerate(hyperparameters_lst):
         n_models = hyperparameters.pop("n_models")
 
-        for model_group in range(n_models):
-            for (i, allele) in enumerate(alleles):
-                print(
-                    "[%2d / %2d hyperparameters] "
-                    "[%2d / %2d replicates] "
-                    "[%4d / %4d alleles]: %s" % (
-                        h + 1,
-                        len(hyperparameters_lst),
-                        model_group + 1,
-                        n_models,
-                        i + 1,
-                        len(alleles), allele))
-
-                train_data = df.ix[df.allele == allele].dropna().sample(
-                    frac=1.0)
-
-                predictor.fit_allele_specific_predictors(
-                    n_models=1,
-                    architecture_hyperparameters=hyperparameters,
-                    allele=allele,
-                    peptides=train_data.peptide.values,
-                    affinities=train_data.measurement_value.values,
-                    models_dir_for_save=args.out_models_dir)
+        inputs = []
+        for (i, allele) in enumerate(alleles[:3]):
+            train_data = df.ix[df.allele == allele].dropna().sample(
+                frac=1.0)
+            inputs.append((n_models,
```

Comment:
In our style we follow the google practice of putting each argument on its own line with one indent, instead of trying to line up multiple lines with the first argument